docs/content/preview/reference/drivers/rust/rust-postgres-reference.md

```diff
@@ -317,28 +320,36 @@ In this case the first two nodes should have 15 connections each, and the third
 After you're done experimenting, run the following command to destroy the local cluster:
 
 ```sh
-./bin/yugabyted destroy --base_dir=$HOME/yugabyte-2.20.0.1/node1
-./bin/yugabyted destroy --base_dir=$HOME/yugabyte-2.20.0.1/node2
-./bin/yugabyted destroy --base_dir=$HOME/yugabyte-2.20.0.1/node3
+./bin/yugabyted destroy --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node1
+./bin/yugabyted destroy --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node2
+./bin/yugabyted destroy --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node3
 ```
 
 ## Configure SSL/TLS
 
 The YugabyteDB Rust smart driver support for SSL is the same as for the upstream driver.
 
-The following table describes the connection parameters required to connect using SSL.
+The following table describes the additional parameters the YugabyteDB Rust smart driver requires as part of the connection string when using SSL.
 
-| Parameter | Description | Default |
-| :-------- | :---------- | :------ |
-| sslmode | Controls usage of TLS. If set to:<br /> <ul><li> disable, TLS will not be used.</li><li> prefer, TLS will be used if available, but not used otherwise.</li><li> require, TLS will be forced to be used.</li></ul> | prefer |
+| rust-postgres parameter | Description | default |
+| :---------------------- | :---------- | :------ |
+| sslmode | SSL Mode | prefer |
+
+rust-postgres supports SSL modes in different ways depending on the driver version, as shown in the following table.
+
+| SSL Mode | Description |
+| :-------- | :---------- |
+| disable | If set, TLS is not used. |
+| prefer (default) | TLS is if available, but not otherwise. |
+| require | If set, TLS is forced to be used. |
 
```

Comment:
Currently, the rust-postgres driver and YugabyteDB Rust smart driver do not support verify-full or verify-ca SSL modes.

and delete the limitations section below